skorecard/bucketers/bucketers.py

```diff
@@ -298,87 +242,35 @@ def __init__(
         self.specials = specials
         self.remainder = remainder
 
-    def fit(self, X, y=None):
-        """Fit X, y."""
-        X = self._is_dataframe(X)
-        self.variables = self._check_variables(X, self.variables)
-        self._verify_specials_variables(self.specials, X.columns)
-
-        features_bucket_mapping_ = {}
-        self.bucket_tables_ = {}
-
-        for feature in self.variables:
+        self.variables_type = "numerical"
 
```

Comment:
do you not need `self.binners: Dict[str, Any] = {}` here?

Reply:
because this bucketer doesn't use any estimators to find the boundaries, so no fitted instances to save. 

Actually, we don't need `self.binners` anywhere, it's not used in any code, just there in case users might want to dig into the fit. Better if we simplify & remove entirely?

Reply:
Ok, went ahead and removed it.